src/vanilla/internals.ts

```diff
@@ -690,9 +687,7 @@ const createStoreHook = (): StoreHook => {
   }
   notify.add = (fn: () => void) => {
     callbacks.add(fn)
-  }
-  notify.delete = (fn: () => void) => {
-    callbacks.delete(fn)
+    return () => void callbacks.delete(fn)
```

Comment:
Prefer block:
```suggestion
    return () => {
      callbacks.delete(fn)
    }
```

Reply:
terser handles it anyway. so, it's just a preference.
<img width="924" alt="image" src="https://github.com/user-attachments/assets/e8f2a885-9d9d-48c3-a0d1-b9e942266e7f" />